cmd/katapult/organizations_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	"github.com/krystal/go-katapult"
+
+	"github.com/krystal/go-katapult/core"
+	"github.com/stretchr/testify/assert"
+)
+
+var organizations = []*core.Organization{
+	{
+		ID:        "loge",
+		Name:      "Loge Enthusiasts",
+		SubDomain: "loge",
+	},
+	{
+		ID:        "testing",
+		Name:      "testing, testing, 123",
+		SubDomain: "test",
+	},
+}
+
+const stdoutOrganizationsList = ` - Loge Enthusiasts (loge) [loge]
+ - testing, testing, 123 (test) [testing]
+`
+
+type mockOrganisationsListClient struct{}
+
+func (mockOrganisationsListClient) List(context.Context) ([]*core.Organization, *katapult.Response, error) {
+	return organizations, nil, nil
+}
+
+func TestOrganizations_List(t *testing.T) {
```

Comment:
I think this should be converted to a tabular test which also tests for when the API response returns an error, or an empty list of organizations, which a lot more likely to happen in the real world compared to the data centers list coming back empty :)